prow/tide/tide.go

```diff
@@ -1823,3 +1844,42 @@ func nonFailedBatchByNameBaseAndPullsIndexFunc(obj runtime.Object) []string {
 
 	return []string{nonFailedBatchByNameBaseAndPullsIndexKey(pj.Spec.Job, pj.Spec.Refs)}
 }
+
+func checkRunNodesToContexts(log *logrus.Entry, nodes []CheckRunNode) []Context {
+	var result []Context
+	for _, node := range nodes {
+		// GitHub gives us an empty checkrun per status context. In theory they could
```

Comment:
Wow that is unexpected and useless behavior... We should consider filing a support ticket to GitHub about that, it seems unlikely to be intentional.